include/glibby/primitives/point3D.h

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
once the other PR goes through, you should remove this file and use the new Point object instead. sorry for the inconvenience :)